SKILL.tsv

```diff
@@ -3883,12 +3883,12 @@ SKILL_20150414_003882	{memo X}{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Gather
 SKILL_20150414_003883	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Use tip of finger to throw coin and attack enemy.
 SKILL_20150414_003884	$Temporarily become invincible from all attacks.
 SKILL_20150414_003885	Create a dispeller that block harmful magic or effects.
-SKILL_20150414_003886	Create your skill into a scroll that anyone can use.
+SKILL_20150414_003886	Craft your skill into a scroll that anyone can use.
 SKILL_20150414_003887	Create a shop to sell your skills.
-SKILL_20150414_003888	Use ritual to heal abnormal state level 2 and above of allies or attack enemies.
+SKILL_20150414_003888	Use a ritual to heal allies with a level 2 abnormal state and above, or damage enemies.
```

Comment:
Use a ritual to heal allies with a level 2 abnormal state and above, or damage enemies. -> Use a ritual to heal allies with a level 2 abnormal state or to inflict damage on enemies.

(There is no 'and above')